### ISSUE TITLE
Live Example not working

### ISSUE THREAD

**Issue by nivalderramas** [mikepricedev/mui-tree-select#3]
When you open the link to the code sandbox, the console gives the following error
```
Error during service worker registration: s@https://codesandbox.io/static/js/sandbox.50af8aaeb.js:1:26358
../common/lib/registerServiceWorker.js/</t.default@https://codesandbox.io/static/js/sandbox.50af8aaeb.js:1:27327
./src/sandbox/index.ts/<@https://codesandbox.io/static/js/sandbox.50af8aaeb.js:1:325941

```

**Comment by mikepricedev** [comment]
Hi @nivalderramas,

Thank you for pointing this out.  I was able to find a [thread](https://github.com/handsontable/hyperformula/issues/1014) that indicated that these errors may have something to do with privacy blockers and codesandbox.io.

I use some privacy features and have seen a "service worker" error in the past, but nothing that kept the demo from loading, so I did not pursue it further.

I was able to load this [example](https://codesandbox.io/s/github/mikepricedev/mui-tree-select/tree/main/example) in the most up to-date versions Chrome, Firefox, and Safari. 

If the problem is persistent you might try relaxing your privacy settings for codesandbox.io and see if that has any effect.

Respectfully,
M Price
